Core/include/Acts/Geometry/ConeVolumeBounds.hpp

```diff
@@ -0,0 +1,228 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/Volume.hpp"
+#include "Acts/Geometry/VolumeBounds.hpp"
+#include "Acts/Utilities/BoundingBox.hpp"
+#include "Acts/Utilities/Definitions.hpp"
+
+#include <cmath>
+
+namespace Acts {
+
+class Surface;
+class CylinderBounds;
+class ConeBounds;
+class RadialBounds;
+class PlanarBounds;
+class Volume;
+
+/// @class ConeVolumeBounds
+///
+/// Volume bound class for describing conical volumes
+/// either with cylindrical inlay or outer boundary, it also allows
+/// for a sectoral description
+class ConeVolumeBounds : public VolumeBounds {
+ public:
+  /// @enum BoundValues for readability
+  enum BoundValues : unsigned int {
+    eInnerAlpha = 0,
+    eInnerOffsetZ = 1,
+    eOuterAlpha = 2,
+    eOuterOffsetZ = 3,
+    eHalfLengthZ = 4,
+    eAveragePhi = 5,
+    eHalfPhiSector = 6,
+    eSize = 7
```

Comment:
The size entry should not have a fixed value so it takes the right value automatically.

Reply:
Ok, changed it consistently.